reSnap.sh

```diff
@@ -103,7 +110,11 @@ fi
 # ssh command
 ssh_host="root@$ip"
 ssh_cmd() {
-  ssh -o ConnectTimeout=1 "$ssh_host" "$@"
+  if [ -n "$ssh_key_file" ]; then
+    ssh -i "$ssh_key_file" -o ConnectTimeout=1 "$ssh_host" "$@"
+  else
+    ssh -o ConnectTimeout=1 "$ssh_host" "$@"
+  fi
```

Comment:
Since there are more options, we should build them.
Otherwise this will lead to a lot of duplicate code.

Maybe something like
```
ssh_options="-o ConnectionTimeout=1"
if [ -n "$ssh_key_file" ]; then
  ssh_options="-i \"$ssh_key_file\" $ssh_options"
fi

ssh_cmd() {
  ssh $ssh_options "$@"
}
```